src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -72,8 +70,36 @@ trait CommonTestData { this: ScalaFutures =>
   val tokenValue = "accessToken"
   val tokenCookie = HttpCookiePair(tokenName, tokenValue)
 
-  val serviceAccountInfo = new ServiceAccountInfo(Option(WorkbenchEmail("testServiceAccount1@example.com")), Option(WorkbenchEmail("testServiceAccount2@example.com")))
+  val clusterServiceAccount = Option(WorkbenchEmail("testClusterServiceAccount@example.com"))
+  val notebookServiceAccount = Option(WorkbenchEmail("testNotebookServiceAccount@example.com"))
+  val serviceAccountInfo = new ServiceAccountInfo(clusterServiceAccount, notebookServiceAccount)
 
+  val auditInfo = AuditInfo(userEmail, Instant.now(), None, Instant.now())
+  def getDataprocInfo(index: Int): DataprocInfo = {
+    DataprocInfo(Option(UUID.randomUUID()), Option(OperationName("operationName" + index.toString)), Option(GcsBucketName("stagingBucketName" + index.toString)), Some(IP("numbers.and.dots")))
+  }
+
+  def getCluster(index: Int): Cluster = {
```

Comment:
Nitpick, but maybe `makeCluster`, `makeDataprocInfo`, etc would be better names since they're making new instances, not returning existing ones.